src/local-fs.ts

```diff
@@ -72,15 +72,19 @@ export class LocalFileSystem implements FileSystem {
                             type: 'fileCreated',
                             fullPath: relPath.split(path.sep).join(pathSeparator),
                             newContent: content
-                        })));
+                        }))
+                        .catch(() => {})
```

Comment:
@amir-arad I am confused, how do you see it? we have strict list of events that FS could emit in API. We can't emit any other..